src/urllib3/poolmanager.py

```diff
@@ -415,11 +443,13 @@ class ProxyManager(PoolManager):
         HTTPS/CONNECT case they are sent only once. Could be used for proxy
         authentication.
 
-    :param _allow_https_proxy_to_see_traffic:
+    :param proxy_ssl_context:
+        The proxy SSL context is used to establish the TLS connection to the
+        proxy when using HTTPS proxies.
+
+    :param _use_forwarding_for_https:
```

Comment:
This parameter should be public. I'd recommend documentation be something like:
```
(Defaults to False) If set to True will forward requests to the HTTPS proxy to be made on behalf of the client instead of
creating a TLS tunnel via the CONNECT method. **Enabling this flag means that
request and response headers and content will be visible from the HTTPS proxy** whereas tunneling
keeps request and response headers and content private.
IP address, target hostname, SNI, and port are always visible to an HTTPS proxy even when this flag is disabled.
```

Reply:
Perfect, I like it!